### ISSUE TITLE
Ctrl-D not working properly in 2.0.1

### ISSUE THREAD

**Issue by xiaket** [jonathanslenders/ptpython#265]
Hi there,

Before the version upgrade, I have set `repl.confirm_exit = False` in `~/.ptpython/config.py` and it was working as intended. However, after the version upgrade, with `repl.confirm_exit = False`, Ctrl-D will not work: ptpython will not do anything about it. If I switch it back to `repl.confirm_exit = True`, Ctrl-D will correctly raise the prompt and work again.

**Comment by xiaket** [comment]
Meanwhile, I'm adding this section in my `config.py` to get it back, but I think it is an unnecessary hack:

```
    @repl.add_key_binding(Keys.ControlD)
    def _(event):
        """
        Ctrl-D to exit.

        This behavior is suppressed somewhere in ptpython.
        """
        event.app.exit(exception=EOFError, style='class:exiting')
```

**Comment by jonathanslenders** [comment]
Fixed here: https://github.com/jonathanslenders/ptpython/commit/ceddae5cf728853b4071c2248d10a349fc8ab5eb

**Comment by jonathanslenders** [comment]
Thanks for reporting this issue!

**Comment by xiaket** [comment]
I can confirm the issue is fixed with ceddae5. Thanks @jonathanslenders ! Great work!
